HW03/anastasia_evsihneyeva/anastasia_evsihneyeva_hw_03_t_03.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  NUMS = /(.*)(?=\ ub)/
+  SPLIT = /[\ :-]/
+  def task3(logs)
+    return '0' if finding_logs(logs).length < 2
+
+    logs = finding_dur(making_time(finding_logs(logs)))
+
+    logs.length == 1 ? logs[0] : logs
+  end
+
+  private
+
+  def finding_logs(logs)
```

Comment:
can we use more descriptive name? for example:
```suggestion
  def core_with_action_logs(logs)
```